SPECS/libxcrypt/libxcrypt.spec

```diff
@@ -138,14 +136,10 @@ Obsoletes:      %{name}-compat < %{version}-%{release}
 %endif
 # We need a version of glibc, that doesn't build libcrypt anymore.
 #Requires:       glibc%{?_isa}         >= %{glibc_minver}
-%if %{with override_glibc}
-# Require a specific glibc version so the post macro is compatible.
-BuildRequires:  glibc-devel = %{glibcversion}
-Requires:       glibc = %{glibcversion}
-%endif
 %if 0%{?fedora} >= 30
 Recommends:     mkpasswd
 %endif
+Conflicts:      glibc-libcrypt
```

Comment:
probably should also add a Requires: glibc >= $VERSION so we make sure the libcrypt file this package installs doesn't conflict with the libcrypt file an older glibc installs (where VERSION is whatever glibc version first includes the glibc-libcrypt split)